docs/preview/features/secret-store/index.md

```diff
@@ -78,6 +77,34 @@ public class HealthController : ControllerBase
 }
 ```
 
+### Using secret store outside .NET hosting
+The secret store is also available directly on the `IServiceCollection` for applications that run outside the .NET hosting but still want to use the Arcus secret store.
+
+Just like you would register the secret store on the `HostBuilder`, you can use the `.AddSecretStore` extension to register the secret store:
+
+```csharp
+IServiceCollection services = ...
+IConfiguration configuration = ...
```

Comment:
Just asking, but why not 'completing' the code sample instead of omitting some things ?

Reply:
Yeah, I added some extra code to make it more clear. Normally, I leave the 'general' types open to not confuse the sample.